examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,29 +14,40 @@
   },
```

Comment:
<p>Add an explanation under the sum here saying that if they do have divergences, they should go read https://docs.pymc.io/pymc-examples/examples/diagnostics_and_criticism/Diagnosing_biased_Inference_with_Divergences.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>